cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casInstitutionLoginView.jsp

```diff
@@ -99,9 +99,9 @@
 
         function institutionLogin () {
 
-            var institutionForm = document.getElementById("institution-form-select");
-            var institutionLoginUrl = institutionForm.options[institutionForm.selectedIndex].value;
-            var selectErrorMessage = document.getElementById("select-error-message");
+            let institutionForm = document.getElementById("institution-form-select");
+            let institutionLoginUrl = institutionForm.options[institutionForm.selectedIndex].value;
+            let selectErrorMessage = document.getElementById("select-error-message");
```

Comment:
Many occurrences: use `let` instead of `var` since there is no longer any compatibility issues with the browsers we support. See https://stackoverflow.com/questions/2356830/what-browsers-currently-support-javascripts-let-keyword.